docs/core/versions/index.md

```diff
@@ -114,7 +118,7 @@ A general Docker tag naming convention is to place the version number before the
 
 The SDK tags should be updated to represent the SDK version rather than Runtime.
 
-It's also possible that we need to fix the .NET Core Tools but reship an existing runtime. In that case, the SDK version is increased (for example, to 2.1.2) and then the Runtime catches up the next time it ships (for example, both the Runtime and SDK ship the following time as 2.1.3).
+It's also possible that the .NET Core Tools need to be fixed but reship an existing runtime. In that case, the SDK version is increased (for example, to 2.1.2) and then the Runtime catches up the next time it ships (for example, both the Runtime and SDK ship the following time as 2.1.3).
```

Comment:
changing the voice from active to passive makes the meaning of reship unclear here. Better might be, "It's also possible that the .NET Core Tools are fixed but reship with an existing runtime" or something similar.
comma before "and then"